jax/linear_util.py

```diff
@@ -237,12 +237,14 @@ def cache(call: Callable):
   Returns:
      A memoized version of ``call``.
   """
+  from jax.config import FLAGS
```

Comment:
Can we move this import to the top of the file?

Reply:
will do